backend/qemu.pm

```diff
@@ -1137,7 +1137,7 @@ sub process_qemu_output {
     my ($buffer) = @_;
     for my $line (split(/\n/, $buffer)) {
         die "QEMU: Shutting down the job" if $line =~ m/key event queue full/;
-        if ($line =~ /^qemu-system-.*: (?!terminating on signal)/) {
+        if ($line =~ /^ qemu-system-.*: (?!terminating on signal)/) {
```

Comment:
Maybe just allow any amount of white-space characters here?

Reply:
I tried and failed :,( Will try again

Reply:
Sorry, I don't know what's wrong. Trying manually `print "COLOR WARN: $line\n" if $line =~ /\s*qemu-system-.*/` works fine but if I use `$line =~ /^\s*qemu-system-.*: (?!terminating on signal)/` in backend/qemu.pm it also matches the line with "terminating on signal"

Reply:
`\s?` should do it

Reply:
that's what I used and stated that it does not work, right?

Reply:
it seems to work in a simple script though:
```perl
@s = (
  "terminating on signal",
  "qemu-system-x: terminating on signal",
  " qemu-system-x: terminating on signal",
  "  qemu-system-x: terminating on signal",
  "qemu-system-x: foo",
  " qemu-system-x: foo",
  "  qemu-system-x: foo",
);
for (@s) {
  say ">>$_<<";
  say 1 if m/^\s*qemu-system-.*: (?!terminating on signal)/
}
__END__
>>terminating on signal<<
>>qemu-system-x: terminating on signal<<
>> qemu-system-x: terminating on signal<<
>>  qemu-system-x: terminating on signal<<
>>qemu-system-x: foo<<
1
>> qemu-system-x: foo<<
1
>>  qemu-system-x: foo<<
1
```

Reply:
@okurz I only saw your "Will try again" comment when I wrote mine. Very strange that it doesn't work.

Reply:
could you take over the changes locally and try where I am doing something wrong?

Reply:
I'll try

Reply:
The problem is the second `:` in the line, e.g.
```
qemu-system-i386: rminating on signal 15 from pid 1500 (/path/to/os-autoinst/t/../isotovideo: backen)
```
So `qemu-system-.*:` matches until the `isotovideo:`

Try:
```
/^\s*qemu-system-[^:]+: (?!terminating on signal)/
```

Reply:
Thanks a lot to both of you